ibis/pyspark/tests/test_basic.py

```diff
@@ -0,0 +1,168 @@
+import pandas as pd
+import pandas.util.testing as tm
+import pytest
+
+import ibis
+
+pytestmark = pytest.mark.pyspark
+
+
+@pytest.fixture(scope='session')
+def client():
+    pytest.importorskip('pyspark')
```

Comment:
This can go at the top of the file, so that there's a skip as early as possible.

Reply:
Moved